training/templates/tensorflow/.gitignore

```diff
@@ -0,0 +1,15 @@
+.idea/
+examples/census-classification/trained_models/
+examples/housing-regression/trained_models/
+examples/babyweight-custom/trained_models/
+examples/german-custom/trained_models/
```

Comment:
With these 4 different template examples, what are they actually showing off / demoing for AI Platform?

Reply:
I guess my point is that all the samples in this repo are supposed to be templates, so what makes these different from the base ones that already exist?